cmd/experimental/kueue-viz/backend/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/akram/kueue-viz-go
+
+go 1.22.0
```

Comment:
Let's use go 1.23.0 to align with the main Kueue

Reply:
done